common/browser_context.go

```diff
@@ -481,3 +481,9 @@ func (b *BrowserContext) runWaitForEventHandler(
 func (b *BrowserContext) getSession(id target.SessionID) *Session {
 	return b.browser.conn.getSession(id)
 }
+
+func (b *BrowserContext) getEvaluateOnNewDocumentSources() []string {
+	sources := make([]string, len(b.evaluateOnNewDocumentSources))
+	copy(sources, b.evaluateOnNewDocumentSources)
+	return sources
+}
```

Comment:
Could you explain the reasoning for the copying here? I suggest adding a comment to explain.

By the way, this commit also makes sense to be in this [commit](4d75cfd3f5a9cc486d1334fc9eb232a94b1d592c) as they both deal with the same behavior: `evaluateOnNewDocument`.